operations/rack_resiliency/manage_critical_services.md

```diff
@@ -0,0 +1,404 @@
+```
+
+Here our list of critical services are present in `rrs-mon-static`.
+
+**To verify if the list is present: **
+
+```bash
+kubectl describe configmap rrs-mon-static -n rack-resiliency
+```
+
+**Example output:**
+
+```text
+Name:         rrs-mon-static
+Namespace:    rack-resiliency
+Labels:       <none>
+Annotations:  <none>
+```
+
+### Data
+
+#### `ceph_pre_monitoring_delay`:
+60
+
+#### `critical-service-config.json`
+
+```text
+{
+  "critical_services": {
+    "cilium-operator": {
+      "namespace": "kube-system",
+      "type": "Deployment"
+    },
+    "coredns": {
+      "namespace": "kube-system",
+      "type": "Deployment"
+    },
+    "cray-power-control-bitnami-etcd": {
+      "namespace": "services",
+      "type": "StatefulSet"
+    },
+    "cray-powerdns-manager": {
+      "namespace": "services",
+      "type": "Deployment"
+    },
+    "cray-sls": {
+      "namespace": "services",
+      "type": "Deployment"
+    },
+    "cray-spire-postgres": {
+      "namespace": "spire",
+      "type": "StatefulSet"
+    },
+    ...
+}
+```
+
+#### `k8s_monitoring_total_time`:
+600
+
+#### `k8s_pre_monitoring_delay`:
+40
+
+#### `ceph_monitoring_polling_interval`:
+60
+
+#### `default_message_level`:
+debug
+
+#### `k8s_monitoring_polling_interval`:
+60
+
+#### `last_updated_timestamp`:
+2025-06-30T04:49:35.298283Z
+
+##### `log_dir`:
+/var/log/rr
+
+#### `unit_of_time`:
+seconds
+
+#### `ceph_monitoring_total_time`:
+600
+
+#### `BinaryData`
+Events:  <none>
+
+**In order to delete the service from the ConfigMap use the following command: **
+
+```bash
+kubectl edit configmap rrs-mon-static -n rack-resiliency
+```
+
+**Example output:**
+
+```text
+apiVersion: v1
+data:
+  ceph_monitoring_polling_interval: "60"
+  ceph_monitoring_total_time: "600"
+  ceph_pre_monitoring_delay: "60"
+  critical-service-config.json: |-
+    {
+      "critical_services": {
+        "cilium-operator": {
+          "namespace": "kube-system",
+          "type": "Deployment"
+        },
+        "coredns": {
+          "namespace": "kube-system",
+          "type": "Deployment"
+        },
+        "cray-activemq-artemis-operator-controller-manager": {
+          "namespace": "dvs",
+          "type": "Deployment"
+        },
+        "cray-capmc": {
+          "namespace": "services",
+          "type": "Deployment"
+        },
+        "cray-ceph-csi-cephfs-provisioner": {
+          "namespace": "ceph-cephfs",
+          "type": "Deployment"
+        },
+        "cray-ceph-csi-rbd-provisioner": {
+          "namespace": "ceph-rbd",
+          "type": "Deployment"
+        },
+    ...
+```
+
+Look for the service you want to delete from the ConfigMap inside `critical-service-config.json` under data section.
+
+**To remove a service:**
+* Locate the service entry you want to remove
+* Delete the entire service block, including the trailing comma
+* Ensure the JSON remains valid
+
+For example Removing a service:<br> Before deleting a service:
+
+```text
```

Comment:
Should be JSON. I'll stop adding these notes for each case, but all JSON or YAML in these docs should be tagged appropriately, unless there is some unavoidable reason not to do so.